parlai/agents/seq2seq/seq2seq.py

```diff
@@ -158,6 +158,12 @@ def add_cmdline_args(argparser):
                            help='Keep replies in the history, or not.')
         agent.add_argument('-pt', '--person-tokens', type='bool', default=False,
                            help='use special tokens before each speaker')
+        agent.add_argument('-ede', '--eval-decode', type=str, default='greedy',
+                           choices=['greedy', 'beam'],
+                           help='Type of search for the best sequence during eval phase')
+        agent.add_argument('--beam-size', type=int, default=-1, help='Beam size if --eval-decode is beam')
+        agent.add_argument('--beam-dump', type=float, default=0.0,
+                           help='The portion of beams to dump from minibatch into <model_file_folder>/beam_dump')
```

Comment:
let's change this to --beam-logfreq and instead make it model_file.beam_dump (append to model file instead of model dir to match how we've done it with other files)